src/strategies/lodestar-vesting/index.ts

```diff
@@ -0,0 +1,48 @@
+import { BigNumberish } from '@ethersproject/bignumber';
+import { formatUnits } from '@ethersproject/units';
+import { Multicaller } from '../../utils';
+import { getAddress } from '@ethersproject/address';
+
+export const author = '0xAppo';
+export const version = '0.1.0';
+
+const abi = [
+  'function balanceOf(address account) external view returns (uint256)'
+];
+
+export async function strategy(
+  space,
+  network,
+  provider,
+  addresses,
+  options,
+  snapshot
+): Promise<Record<string, number>> {
+  const blockTag = typeof snapshot === 'number' ? snapshot : 'latest';
+
+  if (
+    options.beneficiaryAddresses.length >= 25 ||
+    options.contractAddresses.length >= 25
+  ) {
+    throw new Error('Too many beneficiary/contract addresses provided.');
+  }
+  const multi = new Multicaller(network, provider, abi, { blockTag });
+  addresses.forEach((address) => {
+    if (options.beneficiaryAddresses.includes(address)) {
+      const index = options.beneficiaryAddresses.indexOf(address);
```

Comment:
We will include all the voters list in beneficiaryAddresses? it won't work this way. There is a limit for settings length

Reply:
Not sure I understand, this is just meant to check if the voter address in question is one of the beneficiary addresses listed in the options array

Reply:
There is only if condition right, we call the contract only if the address is in benficiaryAddresses? 

Which means there will be maximum of 25 voters only.?

Reply:
If address provided is part of the beneficiary address array listed in the options -> calculate voting power based on vesting tokens in contract. If not -> do nothing. Since the upper limit of beneficiary addresses was added at your request, yes, the maximum allowable voters would be 25. In our case in reality there are only 4.